greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-6288: Additional fields validation to EN Form beta

Ref: https://jira.greenpeace.org/browse/PLANET-6288

This PR adds fields validation to the beta EN Form block.

![Screenshot from 2021-10-06 16-04-20](https://user-images.githubusercontent.com/617346/136225741-a709f8d2-f2df-4549-8b0a-4ffd8ab70631.png)

## Test

- Activate beta blocks
- Add an EN Form beta block to a page

The validation should behave mostly like the current EN Forms.
- Required fields have to be completed
- Only valid email address are accepted.
- No data should be posted to EN as long as there's a validation error.

## Other fixes
- Name of the styles changed to reflect previous names
- Fix line break display for content and form description
- Remove backgorund image interface for style without image
